packages/web3-core/src/web3_request_manager.ts

```diff
@@ -274,6 +274,16 @@ export class Web3RequestManager<
 		response: JsonRpcResponse<ResultType, ErrorType>,
 		{ legacy, error }: { legacy: boolean; error: boolean },
 	): JsonRpcResponse<ResultType> | never {
+		if (isNullish(response)) {
```

Comment:
can you share rpc request and eth node you used for this case?

Reply:
It was tested for `eth_getTransactionReceipt` with ganache for a transaction hash which is not part of the block yet. 

https://github.com/ethereum/execution-apis/blob/main/src/eth/transaction.yaml#L42